src/ListView/ListView.stories.js

```diff
@@ -0,0 +1,62 @@
+import React from 'react'
+import { storiesOf, action } from '@kadira/storybook'
+import { withKnobs, boolean } from '@kadira/storybook-addon-knobs'
+import { defaultTemplate } from '../../storybook/decorators/storyTemplates'
+import MockListManager from './__mocks__/MockListManager'
+import ListView from './ListView'
+import ListViewItem from './ListViewItem'
+import ListViewItemContainer from './ListViewItemContainer'
+import ListViewItemExpansion from './ListViewItemExpansion'
+
+const stories = storiesOf('ListView', module)
+stories.addDecorator(withKnobs)
+stories.addDecorator(
+  defaultTemplate({
+    title: 'List View',
+    documentationLink: 'http://www.patternfly.org/pattern-library/content-views/list-view/'
+  })
+)
+
+stories.addWithInfo(
+  'basic list with expansion',
+  `This is the List View with basic expansion.`,
+  () => <MockListManager />
+)
+
+stories.addWithInfo(
+  'basic list with expansion knobs',
+  `This is the List View with basic expansion.`,
+  () => {
+    const firstLevelExpanded = boolean('First Level Expanded', false)
+    const secondLevelExpanded = boolean('Second Level Expanded', false)
+    return (
+      <ListView>
+        <ListViewItem isActive={firstLevelExpanded}>
+          <ListViewItemContainer
+            isExpansionItem
+            itemClicked={action('first level itemClicked')}
+            isActive={firstLevelExpanded}
+          >
+            First Level
+          </ListViewItemContainer>
+          <ListViewItemExpansion isActive={firstLevelExpanded}>
```

Comment:
@matanwerbner thanks for the review. I considered moving the expansion into the component itself, but this causes two problems:

* it would require us to depend on state (and components should be stateless). An alternative is modifying the children dom automatically, but this is not considered good practice in React. I even got warnings in the console when attempting to modify child nodes in a parent component.
* user will not be allowed to conditionally expand different items as flexibly as they want (say you want to expand item 2 on page load, but not 1 and 3). I know these scenarios seem odd, but they can happen...

thoughts about keeping this stateless @matanwerbner ?

Happy to hear any other suggestions. I started this for my use case, and just wanted to share it as a starting point so community could expand on it...